src/post-submission-actions/program-enrollment-action.ts

```diff
@@ -1,14 +1,13 @@
+import dayjs from 'dayjs';
 import { showToast } from '@openmrs/esm-framework';
 import { createProgramEnrollment, getPatientEnrolledPrograms, updateProgramEnrollment } from '../api/api';
-import dayjs from 'dayjs';
 import { type PostSubmissionAction, type ProgramEnrollmentPayload } from '../types';
-import { useTranslation } from 'react-i18next';
 
 export const ProgramEnrollmentSubmissionAction: PostSubmissionAction = {
   applyAction: async function ({ patient, encounters, sessionMode }, config) {
     const encounter = encounters[0];
     const encounterLocation = encounter.location['uuid'];
-    const { t } = useTranslation();
+    const t = window.i18next.t;
```

Comment:
@ibacher is there a better approach for getting the `t` for use outside of React components? I don't remember. Also, I've not tested whether this works as expected.

Reply:
If you add i18next as a peer dep, you should just be able to do:

```
import { t } from ‘i18next’;
```

Which is cleaner. 

In normal components, we set the default namespace via a I18nextProvider. The raw `t()` function won’t pull from that context, so we’ll need to somehow set the namespace here for things to work as expected; otherwise it’ll always return a null object.

Reply:
@vasharma05 how can I set the namespace here as Ian suggested?